utils/generate_tables.py

```diff
@@ -82,16 +105,19 @@ def __init__(self, simulation_dict):
         self.experiment = simulation_dict["experiment"]
 
         self.ensemble_num = simulation_dict["ensemble_num"]
-        self.cmip_only = simulation_dict["cmip_only"]
+        self.link_type = simulation_dict["link_type"]
 
-        hpss_path = f"/home/projects/e3sm/www/{self.group}/E3SM{self.model_version}/{self.resolution}/{self.simulation_name}"
+        if "hpss_path" in simulation_dict:
+            hpss_path = simulation_dict["hpss_path"]
```

Comment:
If the HPSS path doesn't match our expected form, we're going to just have to get it directly from the csv.